lua/nvim-treesitter/parsers.lua

```diff
@@ -1243,6 +1243,17 @@ list.twig = {
   filetype = "twig",
 }
 
+list.ebnf = {
+  install_info = {
+    url = "https://github.com/ebnf.git",
```

Comment:
you meant to put `https://github.com/RubixDev/ebnf` here

Reply:
Thanks, I thought I just copied it from my personal config...